bats/regression-tests.bats

```diff
@@ -0,0 +1,54 @@
+#!/usr/bin/env bats
+load $BATS_TEST_DIRNAME/helper/common.bash
+
+setup() {
+    setup_common
+}
+
+teardown() {
+    teardown_common
+}
+
+@test "regression-tests: dolt issue #1081" {
+    dolt sql <<SQL
```

Comment:
Any way to pare this down? Pretty hard to see what parts of this large schema are actually relevant to the bug.

Reply:
Also the naming scheme is pretty opaque. Would be nice if a) the set of columns were minimal and b) the table / column names were a bit more meaningful.

Reply:
This is dramatically pared down from the original while retaining the issue. Naming was because it was observed from private  data, and I renamed everything so that we can have a public test that matches the private one. I agree that the names aren't at all useful though.